<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>[*] Graph</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/res/icon.png</normaloff>:/images/res/icon.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>473</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Graph</string>
    </property>
    <addaction name="actionMove"/>
    <addaction name="actionAddVertexes"/>
    <addaction name="actionAddLines"/>
    <addaction name="separator"/>
    <addaction name="actionAdjMatrix"/>
    <addaction name="actionChangeScale"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionMove"/>
   <addaction name="actionAddVertexes"/>
   <addaction name="actionAddLines"/>
   <addaction name="separator"/>
   <addaction name="actionDelete"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/standardbutton-open-32.png</normaloff>:/images/res/standardbutton-open-32.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open graph from file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/standardbutton-save-32.png</normaloff>:/images/res/standardbutton-save-32.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save graph to file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/Exit.png</normaloff>:/images/res/Exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/Move.png</normaloff>:/images/res/Move.png</iconset>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
   <property name="toolTip">
    <string>Move mode</string>
   </property>
  </action>
  <action name="actionAddVertexes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/Create.png</normaloff>:/images/res/Create.png</iconset>
   </property>
   <property name="text">
    <string>Add nodes</string>
   </property>
   <property name="toolTip">
    <string>Add nodes mode</string>
   </property>
  </action>
  <action name="actionAddLines">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/Arrow.png</normaloff>:/images/res/Arrow.png</iconset>
   </property>
   <property name="text">
    <string>Add arrows</string>
   </property>
   <property name="toolTip">
    <string>Add arrows mode</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/Delete.png</normaloff>:/images/res/Delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete selected object</string>
   </property>
  </action>
  <action name="actionAdjMatrix">
   <property name="text">
    <string>Adjacency matrix...</string>
   </property>
   <property name="toolTip">
    <string>Show adjacency matrix</string>
   </property>
  </action>
  <action name="actionChangeScale">
   <property name="text">
    <string>Scale...</string>
   </property>
   <property name="toolTip">
    <string>Change scale</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/file-32.png</normaloff>:/images/res/file-32.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Create a new graph</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadFromFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveToFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDelete</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdjMatrix</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAdjMatrix()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChangeScale</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>changeScale()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>newGraph()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadFromFile()</slot>
  <slot>saveToFile()</slot>
  <slot>changeMode()</slot>
  <slot>deleteSelected()</slot>
  <slot>showAdjMatrix()</slot>
  <slot>changeScale()</slot>
  <slot>newGraph()</slot>
 </slots>
</ui>
